packaging/libstoragemgmt.spec.in

```diff
@@ -200,8 +200,8 @@ libraries.
 Summary:        Files for SMI-S generic array support for %{libstoragemgmt}
 Group:          System Environment/Libraries
 %if 0%{?with_python3}
-BuildRequires:  python3-lmiwbem
-Requires:       python3-lmiwbem
+BuildRequires:  python3-pywbem
+Requires:       python3-pywbem
```

Comment:
Long term we will be moving to pywbem only.  Short term we will be leveraging lmiwbem until a py3 supported version of pywbem is available in fedora.